src/patternfly/components/Chip/chip.scss

```diff
@@ -63,63 +52,39 @@
 
   // Chip overflow modifier
   &.pf-m-overflow {
-    --pf-c-chip--PaddingLeft: var(--pf-c-chip--m-overflow--PaddingLeft);
-    --pf-c-chip--BackgroundColor: var(--pf-c-chip--m-overflow--BackgroundColor);
-
-    // Remove chip border
-    &::before {
-      --pf-c-chip--BorderWidth: var(--pf-c-chip--m-overflow--BorderWidth);
-    }
-
-    .pf-c-button {
-      padding-right: var(--pf-c-chip--m-overflow--c-button--PaddingRight);
-      padding-left: var(--pf-c-chip--m-overflow--c-button--PaddingLeft);
-
-      &::after {
-        border: var(--pf-c-chip--m-overflow--c-button--BorderWidth) solid var(--pf-c-chip--m-overflow--c-button--active--BorderColor);
-        border-radius: var(--pf-c-chip--m-overflow--c-button--BorderRadius);
-      }
+    border: 0;
 
-      &.pf-m-hover,
-      &:hover {
-        // stylelint-disable-next-line
-        &::after {
-          --pf-c-chip--m-overflow--c-button--BorderWidth: var(--pf-c-chip--m-overflow--c-button--hover--BorderWidth);
-          --pf-c-chip--m-overflow--c-button--BorderColor: var(--pf-c-chip--m-overflow--c-button--hover--BorderColor);
-        }
+    &.pf-m-hover,
+    &:hover {
+      &::before {
+        --pf-c-chip--BorderColor: var(--pf-c-chip--m-overflow--hover--before--BorderColor);
       }
+    }
 
-      &.pf-m-active,
-      &:active {
-        // stylelint-disable-next-line
-        &::after {
-          --pf-c-chip--m-overflow--c-button--BorderWidth: var(--pf-c-chip--m-overflow--c-button--active--BorderWidth);
-          --pf-c-chip--m-overflow--c-button--BorderColor: var(--pf-c-chip--m-overflow--c-button--active--BorderColor);
-        }
+    &.pf-m-focus,
+    &:focus {
+      &::before {
+        --pf-c-chip--BorderColor: var(--pf-c-chip--m-overflow--focus--before--BorderColor);
       }
+    }
 
-      &.pf-m-focus,
-      &:focus {
-        // stylelint-disable-next-line
-        &::after {
-          --pf-c-chip--m-overflow--c-button--BorderWidth: var(--pf-c-chip--m-overflow--c-button--focus--BorderWidth);
-          --pf-c-chip--m-overflow--c-button--BorderColor: var(--pf-c-chip--m-overflow--c-button--focus--BorderColor);
-        }
+    &.pf-m-active,
+    &:active {
+      &::after {
+        --pf-c-chip--BorderColor: var(--pf-c-chip--m-overflow--active--before--BorderColor);
       }
     }
   }
 
-  &.pf-m-read-only {
-    padding-top: var(--pf-c-chip--m-read-only--PaddingTop);
-    padding-right: var(--pf-c-chip--m-read-only--PaddingRight);
-    padding-bottom: var(--pf-c-chip--m-read-only--PaddingBottom);
-  }
-
   // Button
   .pf-c-button {
-    padding-right: var(--pf-c-chip--c-button--PaddingRight);
-    padding-left: var(--pf-c-chip--c-button--PaddingLeft);
+    margin-top: var(--pf-c-chip--c-button--MarginTop);
+    margin-right: var(--pf-c-chip--c-button--MarginRight);
+    margin-bottom: var(--pf-c-chip--c-button--MarginBottom);
     font-size: var(--pf-c-chip--c-button--FontSize);
+
+    --pf-c-button--PaddingRight: var(--pf-c-chip--c-button--PaddingRight);
```

Comment:
Can you move these vars to the top of the block? Var declarations should be placed before property declarations.